app/(home)/@posts/page.tsx

```diff
@@ -1,17 +1,20 @@
-import React from 'react';
+import React, { Suspense } from 'react';
 import Post from '@/src/compositions/post/post';
 import { ActionError } from '@/src/models/error.model';
 import { getPosts } from '@/app/actions/post';
 import { PostVariant } from '@/src/compositions/post/types';
+import LoadingPost from './loading';
 
 export default async function Posts() {
     try {
         const paginatedPosts = await getPosts();
         return (
-            <section className="mx-m flex flex-col gap-s">
+            <section className="mx-m flex w-full flex-col gap-s px-m md:w-[680px] md:px-0">
                 {/*<LivePosts />*/}
                 {paginatedPosts.data.map((post) => (
-                    <Post key={post.id} message={post} variant={PostVariant.TIMELINE} />
+                    <Suspense key={post.id} fallback={<LoadingPost />}>
```

Comment:
now we have the suspense and the Loading file - shouldn't we used only one of each for the same page? since it's both on the home page